### ISSUE TITLE
Better a11y of wizard prev/next buttons-

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#4393]
- add `aria-label` attribute: right now the label is just icons which cannot be read
- user can tab to a disabled prev/next button, and the focus changes instantly to somewhere else on the page
    - we should probably allow navigating to a disabled button so the screen reader can be useful

**Comment by avernet** [comment]
[+1 from customer](https://3.basecamp.com/3600924/buckets/1966718/messages/6338049163)
